sorting_test.go

```diff
@@ -257,3 +258,41 @@ func assertRowsEqualByRow[T any](t *testing.T, rowsGot, rowsWant []T) {
 		t.Error(count, "rows mismatched out of", len(rowsWant), "total")
 	}
 }
+
+func TestIssue_82(t *testing.T) {
+	type Record struct {
+		A string `parquet:"a"`
+	}
+
+	fi, err := os.Open("testdata/lz4_raw_compressed_larger.parquet")
+	if err != nil {
+		t.Fatal(err)
+	}
+
+	pr := parquet.NewGenericReader[Record](fi, parquet.DefaultReaderConfig())
+	defer pr.Close()
+
+	var out bytes.Buffer
+
+	pw := parquet.NewSortingWriter[Record](
+		&out,
+		1000,
+		parquet.SortingWriterConfig(
+			parquet.SortingColumns(parquet.Ascending("a")),
+		),
+	)
+
+	if _, err := parquet.CopyRows(pw, pr); err != nil {
+		t.Fatal(err)
+	}
+
+	if err := pw.Close(); err != nil {
+		t.Fatal(err)
+	}
+
+	result := parquet.NewGenericReader[Record](bytes.NewReader(out.Bytes()), parquet.DefaultReaderConfig())
+	defer result.Close()
+	if result.NumRows() != pr.NumRows() {
```

Comment:
Do we want to sort each row group in memory and compare results with the sorting writer? Or do we have other tests which verify that?